ui/desktop/app/routes/cluster-url.js

```diff
@@ -29,10 +29,8 @@ export default class ClusterUrlRoute extends Route {
    * @type {Window}
    */
   get window() {
-    // The Ember way of accessing globals...
-    const document = getOwner(this).lookup('service:-document').documentElement;
     // defaultView === window, but without using globals directly
-    return document.parentNode.defaultView;
+    return this.document.documentElement.parentNode.defaultView;
```

Comment:
I think this can be just returning the window rather than going through all this indirection